prime-router/settings/organizations.yml

```diff
@@ -3226,14 +3230,16 @@
         receiverUrl: http://host.docker.internal:8000/pyas2/as2receive
         receiverId: p1as2
         senderId: CDCPRIMETEST
-    - name: gaen
+    - name: gaen # Note: I don't see this feed in prod
       description: Google/Apple Exposure Notification
       organizationName: wa-phd
       topic: covid-19
       customerStatus: testing
       jurisdictionalFilter:
         - matches(abnormal_flag, A)
         - orEquals(ordering_facility_state, WA, patient_state, WA)
+      routingFilter:
+        - matches(test_result_status, F, S, P, C, X, I)
```

Comment:
the gaen feed doesn't send and results to WA. It only sends a notification message that a positive result has been found along with the demographic information. I think we should talk through which result statuses make sense for that context. Also, I don't think Rick Hawes has moved to prod with that process yet which is why there's no prod feed.

Reply:
Sounds good. I'll take the filter away.